.eslintrc

```diff
@@ -7,7 +7,7 @@
         "node": true
     },
     "parserOptions": {
-        "ecmaVersion": 9
+        "ecmaVersion": 2020
```

Comment:
Currently eslint-plugin-lwc supports [Node >= 10](https://github.com/salesforce/eslint-plugin-lwc/blob/ac84eb9b7659869c0d8b4adbff8c10ebd5976306/package.json#L45). Ecma version `8` maps to `es2018`. 

It would require releasing a major version and bumping the min Node version if we bump the `ecmaVersion` here.  Ideally, we should bump the supported Node version in a separate PR.